.travis.yml

```diff
@@ -15,31 +14,34 @@ before_install:
  - sudo dpkg -i google-chrome*.deb
 before_script:
   - jdk_switcher use oraclejdk8
-# End Selenium-fixes
-
 after_success:
   - npm run build
 
 deploy:
-  - provider: s3
-    acl: public_read
-    access_key_id: $AWS_ACCESS_KEY_ID
-    secret_access_key: $AWS_SECRET_ACCESS_KEY
-    bucket: $AWS_S3_BUCKET
-    upload-dir: $TRAVIS_BRANCH
-    skip_cleanup: true
-    local_dir: dist
-    on:
-      all_branches: true
-  - provider: s3
-    acl: public_read
-    access_key_id: $AWS_ACCESS_KEY_ID
-    secret_access_key: $AWS_SECRET_ACCESS_KEY
-    bucket: $AWS_S3_BUCKET
-    skip_cleanup: true
-    local_dir: dist
-    on:
-      branch: master
+    - provider: s3
+      access_key_id: ${$AWS_ACCESS_KEY_ID}
```

Comment:
IMO, it's better to set this to the actual key value and use `travis encrypt  deploy.secret_access_key` to set the secret_access_key.  This will make it obvious which IAM user/key is used for deployment.